### PR TITLE
Formatting and Typing

### PR DESCRIPTION
**Pull request by Arborym**

### CODE REVIEW

File: Prefix_CopyAll.py [hunk 1]
```diff
@@ -16,11 +16,9 @@ class CopyType(str, Enum):
 def main(
     copy_from: Annotated[Path, Argument(help="The path to copy from")],
     prefix: Annotated[str, Argument(help="the string to search for")],
-    copy_to: Annotated[
```
Comment (zeptofine) [inline Prefix_CopyAll.py:19]: Again, fairly certain that typer does not support this sort of annotation

**Review by zeptofine — COMMENTED** [review]
Very nice. Typer is still not exactly ahead of the curve though, so one small thing can't change

File: Prefix_CopyAll.py (hunk 1 above)
Comment (Arborym) [reply Prefix_CopyAll.py:19]: Fixed

**Comment by Arborym** [comment]
@zeptofine issues have been fixed

**Pull request merged**